react-native/react/reducers/login2.mobile.js

```diff
@@ -59,153 +64,124 @@ const initialState: LoginState = {
   forgotPasswordEmailAddress: '',
   forgotPasswordSubmitting: false,
   forgotPasswordSuccess: false,
-  forgotPasswordError: null
+  forgotPasswordError: null,
+  userPass: {
+    username: '',
+    passphrase: ''
+  },
+  deviceName: {
+    onSubmit: () => {},
+    existingDevices: [],
```

Comment:
should this be null to represent that we haven't fetched the data yet? which is different than no devices?

It might not matter since you'll always probably have one device